website/src/content/development/data-fetching.mdx

```diff
@@ -279,3 +279,27 @@ The available proxy targets are exposed as a `MC_API_PROXY_TARGETS` constant in
 The SDK library does not include features like data normalization, caching, etc. You will need to build these on your own.
 
 </Warning>
+
+<!-- TODO: ### Using your own HTTP client -->
+
+### Using file uploads
+
+In case you need to deal with file uploads, you will probably use [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData).
+
+The `FormData` object can then be sent as the request HTTP body.
```

Comment:
I think it should be send not sent here.

Reply:
I know this sounds weird @tdeekens, but I think "sent" is actually the correct tense in this case.

Reply:
Oh ok. It really felt against my intuition. Thanks for clarifying.